### PR TITLE
added schema changes for LCR

### PR DESCRIPTION
**Pull request by davehorton**

### CODE REVIEW

File: lib/models/lcr-carrier-set-entry.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+const Model = require('./model');
+const {getMysqlConnection} = require('../db');
```
Comment (davehorton) [inline lib/models/lcr-carrier-set-entry.js:2]: We have older code that does it this way 
```
const {getMysqlConnection} = require('../db');
return new Promise((resolve, reject) => {
      getMysqlConnection((err, conn) => {
```
but newer code instead (simpler) does this pattern:
```
const {promisePool} = require('../db');
..
const [r] = await promisePool.query('SELECT *...'
```
For all new code like this, can you follow the latter pattern?  
(When we have time we should do a PR to change all older code as well to this pattern)

File: lib/routes/api/lcrs.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  const {lookupAccountBySid} = req.app.locals;
+  /* account level user can only act on Lcrs associated to the account */
+  if (req.user.hasAccountAuth) {
+    req.body.account_sid = req.user.account_sid;
+  } else if (req.body.account_sid) {
+    const account = await lookupAccountBySid(req.body.account_sid);
+    if (!account) throw new DbErrorBadRequest('unknown account_sid');
+  }
+
+  // check if lcr_carrier_set_entry is vailable
```
Comment (davehorton) [inline lib/routes/api/lcrs.js:19]: typo "available"

File: lib/models/lcr-carrier-set-entry.js (hunk 1 above)
Comment (xquanluu) [reply lib/models/lcr-carrier-set-entry.js]: fixed

File: lib/routes/api/lcrs.js (hunk 2 above)
Comment (xquanluu) [reply lib/routes/api/lcrs.js]: fixed

**Comment by davehorton** [comment]
please update to latest db-helpers and resolve conflicts

**Pull request merged**